packages/near-fast-auth-signer/src/components/AddDevice/AddDevice.tsx

```diff
@@ -201,13 +201,18 @@ function SignInPage() {
         const publicKeyFak = await window.fastAuthController.getPublicKey();
         const existingDevice = await window.firestoreController.getDeviceCollection(publicKeyFak);
         const existingDeviceLakKey = existingDevice?.publicKeys?.filter((key) => key !== publicKeyFak)[0];
+        const user = firebaseAuth.currentUser;
+        // @ts-ignore
+        const oidcToken = user.accessToken;
```

Comment:
Instead of disabling the typescript can we use the `user.getIdToken()` ? 

I logged it a few times on different scenarios and looks like it returns the same token as the accessToken. 

I tried to find the docs for the accessToken and check if it's idd the same as `user.getIdToken()` but I can't find the docs for accessToken